api/test/venues.test.ts

```diff
@@ -16,13 +16,6 @@ describe("GET /venues", () => {
       nextOpeningDates: expectedNextOpeningDates,
     });
   });
-
-  it("returns a 404 if no document for the given ID exists", async () => {
-    const api = mockedApi([]);
-
-    const response = await api.get(`/articles/123`);
-    expect(response.statusCode).toBe(404);
-  });
```

Comment:
Was this removed on purpose?

Reply:
Yes, there is no `/venues/{id}` endpoint. 
Bad copy/paste job when the test was originally added 😅